### PR TITLE
Add some more .eu posts

### PR DESCRIPTION
**Pull request by NickSto**
Trying to port some more content from .eu. The focus here is on posts from multiple subsites, to flesh out the recent news sections of the subsite homepages.

These all seem to be rendering fine, but I could use a double-check that I've categorized them into the right subsites @beatrizserrano

### CODE REVIEW

File: content/news/2022-06-27-gcc2022-webinar-mode/index.md [hunk 1]
```diff
@@ -0,0 +1,43 @@
+---
+title: 'GCC2022: Registration open for Webinar mode'
+tease: "For those that won't attend in person, the GCC Organizing Committee is happy to announce a virtual webinar-like option"
+date: "2022-06-27"
+author: beatrizserrano
+subsites: [global, eu, pasteur, freiburg, erasmusmc, elixir-it, belgium, genouest]
```
Comment (beatrizserrano) [inline content/news/2022-06-27-gcc2022-webinar-mode/index.md:6]:
```suggestion
subsites: [global, all-eu]
```

File: content/news/2022-07-06-paper-tomas-teaching/index.md [hunk 2]
```diff
@@ -0,0 +1,25 @@
+---
+title: 'New Paper "Creating a good learning and sharing environment for bioinformatics"'
+date: "2022-07-06"
+tease: "Tomas Klingström and J.I. Ohlsson have published a paper about the use of BioContainers, Galaxy, and the GTN for bioinformatics researchers."
+hide_tease: true
+tags:
+- paper
+- citations
+supporters:
+subsites: [global, eu, pasteur, freiburg, erasmusmc, elixir-it, belgium, genouest]
```
Comment (beatrizserrano) [inline content/news/2022-07-06-paper-tomas-teaching/index.md:10]:
```suggestion
subsites: [global, all-eu]
```

File: content/news/2022-08-10-open-admin-position/index.md [hunk 3]
```diff
@@ -0,0 +1,62 @@
+---
+title: The European Galaxy Team is hiring!
+date: "2022-08-10"
+tease: "The Freiburg Galaxy Team is looking for a Systems Engineer/DevOps Engineer!"
+hide_tease: true
+location: Freiburg, Germany
+tags: [jobs]
+subsites: [global, eu, pasteur, freiburg, erasmusmc, elixir-it, belgium, genouest]
+main_subsite: eu
```
Comment (beatrizserrano) [inline content/news/2022-08-10-open-admin-position/index.md:9]:
```suggestion
main_subsite: freiburg
```

**Comment by NickSto** [comment]
Not sure why the test's failing. It's working when I run it locally. It might be some intermittent issues with Gitter.

**Comment by NickSto** [comment]
Okay, it passed now. Probably was on Gitter's side.

**Pull request merged**